aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/alarmConfigurations.ts

```diff
@@ -0,0 +1,101 @@
+import {ComparisonOperator} from '@aws-sdk/client-cloudwatch';
+
+import modelDescriptions from '../../../../../../apps/static/aichat/modelDescriptions.json';
```

Comment:
nit: mind adding a TODO that we want to remove this dependency if possible (i.e. try to use shared_constants.rb somehow?)